### PR TITLE
Rebase 3.32 (debian packaging)

### PR DESCRIPTION
**Pull request by uajain**
https://phabricator.endlessm.com/T25164

### CODE REVIEW

File: debian/control.in [hunk 1]
```diff
@@ -3,63 +3,69 @@ Section: gnome
...
+               libgexiv2-dev (>= 0.10.0),
+               libgstreamer-plugins-base1.0-dev,
+               libgirepository1.0-dev (>= 0.10.7-1~),
+               libglib2.0-dev (>= 2.49.1),
+               libglib2.0-doc,
                libgnome-autoar-0-dev (>= 0.2.1),
                libgnome-desktop-3-dev (>= 3.10),
+               libgtk-3-dev (>= 3.22.27),
+               libgtk-3-doc,
+               eos-shell-content-dev,
```
Comment (GeorgesStavracas) [inline debian/control.in:24]: Since you're on it, please move this package according to the alphabetical order.

File: debian/rules [hunk 2]
```diff
@@ -1,47 +1,44 @@
...
 override_dh_auto_configure:
 	dh_auto_configure -- \
-		-Denable-packagekit=true \
-		-Denable-desktop=true \
-		-Denable-exif=true \
-		-Denable-gtk-doc=true \
-		-Denable-packagekit=true \
-		-Denable-nst-extension=true \
-		-Denable-unity=true \
-		-Denable-xmp=true \
```
Comment (GeorgesStavracas) [inline debian/rules:28]: Paranoid question: did you make sure we are not missing any of these flags?

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]

File: debian/rules (hunk 2 above)
Comment (uajain) [reply debian/rules:28]: Not sure I follow. Can you explain a bit more? 

(This is coming from upstream debian packaging and I only have a little idea of what it does, If you can give me the specifics, I can look into it.)

Comment (GeorgesStavracas) [reply debian/rules:28]: Did you double check that none of the flags that were dropped are needed by us?

**Comment by wjt** [comment]
We have moved this package to OBS.

**Pull request closed without merging**